src/clib/pioc_support.c

```diff
@@ -4951,3 +4923,15 @@ int __wrap_ADIOI_Type_create_hindexed_x(int count,
     return ret;
 }
 #endif
+
+char *spio_strdup(const char *str)
+{
+    size_t n = strlen(str) + 1;
```

Comment:
In the PR that handles the return value from spio_strdup() also don't forget to handle the "str == NULL" case here